code_challenges.rb

```diff
@@ -3,49 +3,113 @@
 
 # --------------------1) Create a method that takes in an array and returns an array with only odd numbers sorted from least to greatest.
 
+# i was able to get the 2 methods that do this but I couldnt figure out how to put them together i tried a few different ways but wasnt able to get them to work. 
+
+def only_numbers array
+    array.select { |element| element.is_a?(Integer) }.sort().filter{ |element| element.odd? }
```

Comment:
select and filter are the same method, consider a way to combine the conditional statements